spec/std/file_spec.cr

```diff
@@ -648,6 +648,59 @@ describe "File" do
     end
   end
 
+  describe "append" do
+    it "can create a new file" do
+      filename = "#{__DIR__}/data/temp_append.txt"
+      begin
+        File.write(filename, "hello", mode: "a")
+        File.read(filename).should eq("hello")
+      ensure
+        File.delete(filename)
+      end
+    end
+
+    it "can append to an existing file" do
+      filename = "#{__DIR__}/data/temp_append.txt"
+      begin
+        File.write(filename, "hello", mode: "a")
+        File.read(filename).should eq("hello")
+        File.write(filename, " world", mode: "a")
+        File.read(filename).should eq("hello world")
+      ensure
+        File.delete(filename)
+      end
+    end
+
+    it "appends bytes" do
```

Comment:
This spec seems unnecessary, it doesn't do anything else than what is specced for `IO`/`File` anyway.